dapps/digital-twin-detail/src/i18n/de.json

```diff
@@ -0,0 +1,34 @@
+{
+  "_twin-detail": {    
+    "data": {
+      "data-title": "Daten",
+      "general": {
+        "desc": "Beschreibung",
+        "general-title": "Allgemein",
+        "information-title": "Informationen",
+        "name": "Name",
+        "owner": "Besitzer",
+        "twin-image": "Zwilling Bild",
+        "type": "Typ"
+      },
+      "logs": {
+        "logs-title": "Logs"
+      },
+      "specifications": {
+        "specifications-title": "Spezifikationen"
+      }
+    },
+    "did": {
+      "did-title": "Decentralized ID"
+    },
+    "overview": {
+      "overview-title": "Übersicht"
+    },
+    "sharings": {
+      "sharings-title": "Sharings"
```

Comment:
for the "Share" button we use currently "freigeben". Analogue it would be "Freigaben" but maybe we find a better wording for it. Maybe also use english here in general.